src/packages/migrate/src/utils/seed.ts

```diff
@@ -67,6 +67,29 @@ export function detectSeedFiles(schemaPath) {
   return detected
 }
 
+function getSeedScript(type: 'TS' | 'JS', seedFilepath: string) {
+  let script = `
+  const __seed = require('./${seedFilepath}')
+  const __keys = Object.keys(__seed)
+
+  // Execute default or "seed" named export
+  if (__keys && __keys.length) {
+    if (__keys.indexOf('default') !== -1) {
+      __seed.default()
+    } else if (__keys.indexOf('seed') !== -1) {
+      __seed.seed()
+    }
+  } else {
+    ''
+  }`
+
+  if (type === 'TS') {
+    script = 'declare const require: any' + script
```

Comment:
Do you also need to add `declare const module: any` or is that not applicable?

Reply:
Only require is used in this script so no need to declare a type for module global var